packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/components/NftAssetItem.tsx

```diff
@@ -124,17 +123,21 @@ const NftAssetItem: React.FC<Props> = ({ asset }) => {
               background='linear-gradient(127.95deg, #DADADA 0%, #F4F7FB 0.01%, #2F7CF6 100%)'
             />
           )}
-          <NftAssetImageType
-            top='10px'
-            right='10px'
-            animation_url={asset.animation_url}
-            image_url={image}
-          />
         </Link>
       </LinkContainer>
       <AssetDetails>
         <Flex flexDirection='column' gap={8}>
-          <Text size='18px' style={{ marginTop: '4px' }} color='white' weight={600}>
+          <Text
+            size='18px'
+            style={{
```

Comment:
Too many inline styles, a good practice for code readability would be to move to a styled component